osgp/platform/osgp-secret-management/src/main/java/org/opensmartgridplatform/secretmanagement/application/endpoints/SecretManagementEndpoint.java

```diff
@@ -37,86 +45,158 @@
 @Slf4j
 public class SecretManagementEndpoint {
 
-    private static final String NAMESPACE_URI = "http://www.opensmartgridplatform"
-            + ".org/schemas/security/secretmanagement";
+    private static final String NAMESPACE_URI =
+            "http://www.opensmartgridplatform" + ".org/schemas/security/secretmanagement";
```

Comment:
You could argue about how nice it looked before, but splitting a string in two like this, when on a single line, looks even worse to me.